core/src/main/java/org/testcontainers/Testcontainers.java

```diff
@@ -1,8 +1,12 @@
 package org.testcontainers;
 
-import lombok.experimental.UtilityClass;
+import java.util.Map;
+import java.util.Map.Entry;
+
 import org.testcontainers.containers.PortForwardingContainer;
 
+import lombok.experimental.UtilityClass;
```

Comment:
the import order is wrong. Should be before `import org.testcontainers.containers.PortForwardingContainer;`

Reply:
done